### ISSUE TITLE
More dark mode

### ISSUE THREAD

**Issue by sanarena** [GeekMagicClock/smalltv-pro#98]
It would be very (very very) useful to have another theme, clone of PRICE NUMBERS ONLY  for bitcoin where there is no background color. 
I do not mind green/red status of the price, but I would like to know what is the current price. even in darkness.
Currently I am facing a problem where background light make reading number very hard specially if there is a bit of distance.
If there is only the number, without background light, the number would be more visible and easier to read.

**Comment by GeekMagicClock** [comment]
@sanarena  Hello, if you turn the  screen on, the background light will be on, that's why we can see the display. So I guess you mean price's backgroud color? Turn the background color to black ?

**Comment by sanarena** [comment]
Yes, exactly. background light of the LCD is okay. but background color of bitcoin price in fullscreen can be removed to enhance dark mode.

**Comment by eribertomota** [comment]
Hi @GeekMagicClock ! I agree with @sanarena .

Could you add an extra option in Theme Style in Bitcoin section to remove the background red/green color?

Thanks a lot for your work and nice improvements.
